introduction.md

```diff
@@ -0,0 +1 @@
+Hi! This is a longer introduction about myself including an interesting and thought provoking biography of my life and accomplishments.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.